components/global/global_header.scss

```diff
@@ -38,12 +38,21 @@
 
             // adjust height of team sidebar
             height: calc(100% - 40px);
+
+            body.announcement-bar--fixed & {
+                height: calc(100% - 80px);
```

Comment:
since the webapp renders filling the full viewport and adding specific scroll to internal elements, wouldn't it make more sense  to use vh instead of percentage for this calculation?

Reply:
@pablovelezvidal is this ok as is for now?